react/modules/useEffectSkipFirstRender.ts

```diff
@@ -0,0 +1,17 @@
+import { useRef, useEffect } from 'react'
+
+export const useEffectSkipFirstRender = (
+  fn: React.EffectCallback,
+  deps: React.DependencyList
```

Comment:
You didn't import the React

```suggestion
import { useRef, useEffect, EffectCallback, DependencyList } from 'react'

export const useEffectSkipFirstRender = (
  fn: EffectCallback,
  deps: DependencyList
```